home.html

```diff
@@ -62,6 +62,7 @@
        <!-- start introduction-->
        <div class="intro-container">
          <h1 id="Introduction">Introduction</h1>
+         <p>HTML personally i describ it as thee backbone of website development</p>

```

Comment:
By writing `describ` did you mean `describe`? And by writing `thee` did you mean `the`? If so change them and push your changes